### DISCUSSION TITLE
[Suggestion] Shake to reset also adds a new bookmark

### DISCUSSION THREAD

**Discussion by ncoll36** [PaulWoitaschek/Voice#1298] [Ideas]
I like the whole shake to reset the sleep timer. When I'm listening to the book while trying to sleep I usually like to set the timer to small increments so it's easy to find my position later on. Only problem is that I need to unlock my phone and reset the timer each time.

So I'm proposing that the shake to reset feature also creates a new bookmark from the time you shake your phone.

**Comment by test2a** [comment]
This.

I listen to books during going to sleep and this is one troubling thing for when we loose the position and then we have to go and manually find the last position by hit and trial.

**Comment by RichyHBM** [comment]
Adding myself as someone that would like this feature.

If you're trying to sleep but can't quite manage to it's nice to be able to reset without turning on the screen, but not having it remember the last shaken position makes getting back to that point annoying
